package.json

```diff
@@ -147,6 +148,7 @@
     "stylelint-scss": "^4.2.0",
     "ts-node": "^10.8.2",
     "typescript": "^4.9.5",
+    "underscore": "^1.13.6",
```

Comment:
In our doc site package, we're actually using `lodash` instead of `underscore`. Should we use the same package?

Reply:
Oh good call! The script actually works exactly the same using lodash with no changes 🥳 So I'm going to remove the underscore dependency.